pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2193,6 +2314,18 @@ func buildCompleteFilterChain(mutable *istionetworking.MutableObjects, opts buil
 			mutable.Listener.FilterChains[i].Filters = append(mutable.Listener.FilterChains[i].Filters, filter)
 			log.Debugf("attached Thrift filter with %d thrift_filter options to listener %q filter chain %d",
 				len(thriftProxies[i].ThriftFilters), mutable.Listener.Name, i)
+		} else if opt.dubboOpts != nil && features.EnableDubboFilter {
+			// Add the TCP filters first.. and then the Dubbo filter
```

Comment:
delete code. does not matter for dubbo/thrift

Reply:
Unlike mysql/mangodb, Dubbo filter handles the routing by itself instead of passing it through to a tcp proxy filter.